app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,13 @@
+// "bootstrap-sprockets" must be imported before "bootstrap" and "bootstrap/variables"
+@import "bootstrap-sprockets";
+@import "bootstrap";
+@import "font-awesome";
+@import "bootstrap_and_overrides";
+@import "select2";
+@import "select2-bootstrap";
+@import "token-input-bootstrappy";
+@import "bootstrap-datepicker3";
+@import "list.unlist";
+@import "list.missing";
+@import "recurring_select";
+@import "actiontext";
```

Comment:
Just a question (I haven't tried this out in foodsoft):

Wouldn't it be better to use `@use` and/or `@forward` now  wherever possible? 
see https://sass-lang.com/blog/import-is-deprecated/

I guess for old libs such as bootstrap this will not work and we'll have to stick with `@import`, but for the own internal files this should be comparatively easy, right?

Reply:
Ah, I see. Thanks for explaining!